### PR TITLE
uptade wikimedia-ui-base dependency to v0.18

### PR DESCRIPTION
**Pull request by SaiSan-WMDE**
A couple of new tokens were introduced in order to align our styles with wikimedia-ui-base. No values were modified in this update.

Issue: https://phabricator.wikimedia.org/T268032

### CODE REVIEW

**Review by micgro42 — DISMISSED** [review]

**Review by itamargiv — CHANGES_REQUESTED** [review]
Hey hey, now that we have a CHANGELOG, can you add a line to the log explaining that we added new tokens?

File: CHANGELOG.md [hunk 1]
```diff
@@ -19,6 +19,7 @@ development)_
 - New Message Component
 - Initial version of the Lookup component
 - Additional icons to the Icon component
+- New tokens based on wikimedia-ui-base v.0-18
```
Comment (SaiSan-WMDE) [inline CHANGELOG.md:22]: is this too generic, @itamargiv? Should I specify which tokens were added?

Comment (SaiSan-WMDE) [inline CHANGELOG.md:22]:
```suggestion
- New tokens based on wikimedia-ui-base v 0.18
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -19,6 +19,7 @@ development)_
 - New Message Component
 - Initial version of the Lookup component
 - Additional icons to the Icon component
+- New tokens based on wikimedia-ui-base v 0.18
```
Comment (SaiSan-WMDE) [inline CHANGELOG.md:22]:
```suggestion
- New tokens based on wikimedia-ui-base v 0.18.0
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -19,6 +19,7 @@ development)_
 - New Message Component
 - Initial version of the Lookup component
 - Additional icons to the Icon component
+- New tokens based on wikimedia-ui-base v 0.18.0
```
Comment (SaiSan-WMDE) [inline CHANGELOG.md:22]: is this too generic, @itamargiv ? should I specify which tokens were added?

Comment (itamargiv) [reply CHANGELOG.md:22]: Nah, I think this is good for now. Thank you :)

**Review by itamargiv — APPROVED** [review]

**Pull request merged**